cli/azd/pkg/azureutil/resource_operation.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azureutil
+
+import (
+	"time"
+)
+
+type DeployedAzureResource struct {
+	Id                      string
+	ResourceType            string
+	ResourceTypeDisplayName string
+	ResourceName            string
+	DeployedTimestamp       time.Time
+}
```

Comment:
I feel like we should avoid duplicating the field from `tools.AzCliResourceOperation` in here.

Have you consider adding `ResourceTypeDisplayName` to `tools.AzCliResourceOperation` instead of creating this extra structure?

Or another option might be to compose `tools.AzCliResourceOperation` here.

Basically, we can save making the string copies from the operations slice into this other slice.